app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentRow.kt

```diff
@@ -37,20 +37,27 @@ data class OverdueAppointmentRow(
     val isAtHighRisk: Boolean,
     val lastSeenDate: String,
     val diagnosedWithDiabetes: Answer?,
-    val diagnosedWithHypertension: Answer?
+    val diagnosedWithHypertension: Answer?,
+    val showDiagnosisLabel: Boolean
 ) : ItemAdapter.Item<UiEvent> {
 
   companion object {
 
     fun from(
         appointments: List<OverdueAppointment>,
         clock: UserClock,
-        dateFormatter: DateTimeFormatter
+        dateFormatter: DateTimeFormatter,
+        showDiagnosisLabel: Boolean
     ): List<OverdueAppointmentRow> {
-      return appointments.map { overdueAppointment -> from(overdueAppointment, clock, dateFormatter) }
+      return appointments.map { overdueAppointment -> from(overdueAppointment, clock, dateFormatter, showDiagnosisLabel) }
     }
 
-    private fun from(overdueAppointment: OverdueAppointment, clock: UserClock, dateFormatter: DateTimeFormatter): OverdueAppointmentRow {
+    private fun from(
+        overdueAppointment: OverdueAppointment,
+        clock: UserClock,
+        dateFormatter: DateTimeFormatter,
+        showDiagnosisLabel: Boolean
```

Comment:
Since showing/hiding the diagnosis label is a presentation detail, can we rename this parameter to `diabetesManagementEnabled` instead?